gytheio-commons/pom.xml

```diff
@@ -14,7 +14,7 @@
         <dependency.commons-logging.version>1.1.2</dependency.commons-logging.version>
         <dependency.commons-io.version>2.6</dependency.commons-io.version>
         <dependency.commons-lang.version>2.6</dependency.commons-lang.version>
-        <dependency.tika.version>1.5</dependency.tika.version>
+        <dependency.tika.version>1.20</dependency.tika.version>
```

Comment:
FYI: ACS 6.1 / ATS 1.0 still use 1.17-20180201-alfresco-patched

Reply:
Yes, I figured those could exclude these dependencies and bring in their own.

Reply:
According to GitHub's [security alert](https://github.com/Alfresco/gytheio/network/alert/gytheio-commons/pom.xml/org.apache.tika:tika-core/open) we need to update to Tike 1.19.1 or later, so I'm not sure we can align this one to the version the Alfresco repo is using.

Reply:
Since ACS and ATS are using, the same, custom fork of Tika (https://github.com/Alfresco/alfresco-tika/tree/1.17-alfresco-patched) I don't think this should be a concern here. As @rgauss points out they are already excluding this dependency from their own usage. That would continue here.  I'll open a new issue since there are a number of CVE's that have been opened against Tika since our fork at 1.17 (https://tika.apache.org/security.html)